.github/workflows/benchmark.yml

```diff
@@ -146,3 +146,113 @@ jobs:
         LC_ALL=C sort CredSweeper/cicd/benchmark.txt | diff - benchmark.txt
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+  performance_benchmark:
+    # put the benchmark in single job to keep constant environment during test
+    needs: [download_data]
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+
+    - name: Checkout CredData
+      uses: actions/checkout@v3
+      with:
+        repository: Samsung/CredData
+
+    - name: Cache data
+      id: cache-data
+      uses: actions/cache@v3
+      with:
+        path: data
+        key: cred-data-${{ hashFiles('snapshot.yaml') }}
+
+    - name: Failure in case when cache missed
+      if: steps.cache-data.outputs.cache-hit != 'true'
+      run: exit 1
+
+    - name: Exclude very huge data
+      if: steps.cache-data.outputs.cache-hit == 'true'
+      run: rm -vrf data/81cd05d0
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Update PIP
+      run: python -m pip install --upgrade pip
+
+    - name: Install released CredSweeper
+      run: python -m pip install credsweeper
+
+    - name: Run performance benchmark
+      run: |
+        credsweeper --path requirements.txt --banner
```

Comment:
Why is it needed? `requirements.txt` file scanning?

Reply:
It's time to do --banner self-sufficient!

Reply:
Thank you for fix!